ces_client/utils.py

```diff
@@ -32,15 +32,18 @@ def create_msg_from_ben(self, msg):
             return 'Oh.', answers
 
         state, closing_state = self.find_user_state(msg)
+
         if not closing_state:
-            text = state.message.text
+            text = state.message.text.replace('\n', '<br>')
```

Comment:
this is one of the things you are doing in the method. it seem redundant?

Reply:
oh, whoops! that's leftover.